cmd/scheduler/app/options/options.go

```diff
@@ -85,6 +86,7 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	// volcano scheduler will ignore pods with scheduler names other than specified with the option
 	fs.StringArrayVar(&s.SchedulerNames, "scheduler-name", []string{defaultSchedulerName}, "vc-scheduler will handle pods whose .spec.SchedulerName is same as scheduler-name")
 	fs.StringVar(&s.SchedulerConf, "scheduler-conf", "", "The absolute path of scheduler configuration file")
+	fs.StringVar(&s.SchedulerConfigmap, "scheduler-configmap", "volcano-scheduler-configmap ", "The configmap of scheduler configuration")
```

Comment:
So what's the difference bewteen `SchedulerConf` and `SchedulerConfigmap`?  Besides, how about that user remove the configmap by fault?

Reply:
- `SchedulerConf`is the file which kubelet mount from `SchedulerConfigmap`, their contents are same. But the difference is that watching `SchedulerConfigmap` change by informer will make configuration modification take effect asap, comparing to filewatch. As  # 2332 mentioned,
> As a result, the total delay from the moment when the ConfigMap is updated to the moment when new keys are projected to the pod can be as long as kubelet sync period (1 minute by default) + ttl of ConfigMaps cache (1 minute by default) in kubelet.

- If user remove the configmap by fault, then scheduler will use the default or the old configuration. IMO, it's no risks.

Reply:
> * `SchedulerConf`is the file which kubelet mount from `SchedulerConfigmap`, their contents are same. But the difference is that watching `SchedulerConfigmap` change by informer will make configuration modification take effect asap, comparing to filewatch. As  # 2332 mentioned,

IC. If that, I think this parameter is not necessary. Let's keep the `schedulerConf`. And users should not configure twice.




> * If user remove the configmap by fault, then scheduler will use the default or the old configuration. IMO, it's no risks.

OK. That's reasonable.

Reply:
Maybe we should follow a deprecated policy here, keep both `schedulerConf` and `schedulerConfigmap`, then we should:
1) add warning messages to `schedulerConf` to notify end-users that it's deprecated and will be removed in the future
2) add validations that `schedulerConf` and `schedulerConfigmap` can't be enabled at the same time
3) choose the implement by the flag set by end-user, if so, we should keep both the implements

This is backwards compatible and will not break end-users.